test_sets/smoke_tests_non_media.txt

```diff
@@ -0,0 +1,9 @@
+./test_scripts/Smoke/Policies/001_PTU_all_flows.lua
```

Comment:
@GetmanetsIrina Propose to change numbering in script's names.
Think all of them can be placed at the end of test set, so the numbers will be: 038 - 045